pool-async/src/main/java/com/github/jasync/sql/db/pool/ActorBasedObjectPool.kt

```diff
@@ -337,17 +336,6 @@ private class ObjectPoolActor<T : PooledObject>(
         }
     }
 
-    private fun checkItemsInCreationForTimeout() {
```

Comment:
I would keep that as a safety mechanism, maybe with a bigger threshold like createTimeout * 2 (or + 20 seconds)

Reply:
Sounds like a good idea to keep things backwards compatible. I have restored it.